<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaletteTitleBar</class>
 <widget class="QWidget" name="PaletteTitleBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>32</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowminimumheight="1,1,1,0,0">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="3" column="0">
    <widget class="Qt::WidgetsEx::OrientableLabel" name="lblTitle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblIcon">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="Qt::WidgetsEx::HoverButton" name="pBtnFold">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="Qt::WidgetsEx::HoverButton" name="pBtnClose">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Qt::WidgetsEx::HoverButton" name="pBtnReduce">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Qt::WidgetsEx::HoverButton</class>
   <extends>QPushButton</extends>
   <header>hoverbutton.h</header>
  </customwidget>
  <customwidget>
   <class>Qt::WidgetsEx::OrientableLabel</class>
   <extends>QLabel</extends>
   <header>orientablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="E:/MediaCadQt/MdWidgets/mdwidgets.qrc"/>
 </resources>
 <connections/>
</ui>
